phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -141,7 +137,8 @@ public void setOnRetryButtonClicked(EventHandler<? super MouseEvent> event) {
     private void showAppDetails(ApplicationDTO application, SettingsManager settingsManager) {
         final AppPanel appPanel = new AppPanel(application, themeManager, settingsManager);
         appPanel.setOnScriptInstall(this::installScript);
-        showRightView(appPanel);
+        appPanel.setMaxWidth(400);
```

Comment:
If we always want to limit the size of the right view, it may be a good idea to move this to the `showDetailsView` method.